pages/deployment/wifi.md

```diff
@@ -86,7 +86,7 @@ In general the network config follows the [ConnMan][connman] configuration file
 
 ## Changing your Network Configuration
 
-On all devices excluding the [Intel Edison](/installing/gettingStarted-Edison), it is possible to change your wifi SSID or Passphrase after downloading the `.img`.
+It is possible to change your wifi SSID or Passphrase after downloading the `.zip`, however it is not possible to change the wifi SSID and passphrase of devices that run from internal eMMC without reflashing the device.
```

Comment:
I would just say "downloading the image". Also, the word "passphrase" is capitalised once, and then all lower-case.